modules/connectivity/locals.tf

```diff
@@ -222,6 +229,13 @@ locals {
     local.deploy_virtual_hub[location] &&
     virtual_hub.config.vpn_gateway.enabled
   }
+  deploy_virtual_hub_azure_firewall_policy = {
+    for location, virtual_hub in local.virtual_hubs_by_location :
+    location =>
+    local.deploy_virtual_hub[location] &&
+    virtual_hub.config.azure_firewall.enabled &&
+    length(try(local.custom_settings.azurerm_firewall["virtual_wan"][location].firewall_policy_id, "")) == 0
```

Comment:
Again, if the try fails does the length comparison work?

Reply:
as above